src/main/java/org/apache/sysml/hops/estim/EstimatorMatrixHistogram.java

```diff
@@ -27,182 +27,178 @@
 		long nnz = 0;
-		//special case, with exact sparsity estimate, where the dot product
-		//dot(h1.cNnz,h2rNnz) gives the exact number of non-zeros in the output
-		if( h1.rMaxNnz <= 1 || h2.cMaxNnz <= 1 ) {
-			for( int j=0; j<h1.getCols(); j++ )
+		// special case, with exact sparsity estimate, where the dot product
+		// dot(h1.cNnz,h2rNnz) gives the exact number of non-zeros in the output
+		if (h1.rMaxNnz <= 1 || h2.cMaxNnz <= 1) {
+			for (int j = 0; j < h1.getCols(); j++)
 				nnz += h1.cNnz[j] * h2.rNnz[j];
 		}
-		//special case, with hybrid exact and approximate output
-		else if(h1.cNnz1e!=null && h2.rNnz1e != null) {
-			int mnOut = h1.getRows()*h2.getCols();
+		// special case, with hybrid exact and approximate output
+		else if (h1.cNnz1e != null && h2.rNnz1e != null) {
+			int mnOut = h1.getRows() * h2.getCols();
 			double spOutRest = 0;
-			for( int j=0; j<h1.getCols(); j++ ) {
-				//exact fractions, w/o double counting 
+			for (int j = 0; j < h1.getCols(); j++) {
+				// exact fractions, w/o double counting
 				nnz += h1.cNnz1e[j] * h2.rNnz[j];
-				nnz += (h1.cNnz[j]-h1.cNnz1e[j]) * h2.rNnz1e[j];
-				//approximate fraction, w/o double counting
-				double lsp = (double)(h1.cNnz[j]-h1.cNnz1e[j]) 
-					* (h2.rNnz[j]-h2.rNnz1e[j]) / mnOut;
-				spOutRest = spOutRest + lsp - spOutRest*lsp;
+				nnz += (h1.cNnz[j] - h1.cNnz1e[j]) * h2.rNnz1e[j];
+				// approximate fraction, w/o double counting
+				double lsp = (double) (h1.cNnz[j] - h1.cNnz1e[j]) * (h2.rNnz[j] - h2.rNnz1e[j]) / mnOut;
+				spOutRest = spOutRest + lsp - spOutRest * lsp;
 			}
-			nnz += (long)(spOutRest * mnOut);
+			nnz += (long) (spOutRest * mnOut);
 		}
-		//general case with approximate output
+		// general case with approximate output
 		else {
-			int mnOut = h1.getRows()*h2.getCols();
+			int mnOut = h1.getRows() * h2.getCols();
 			double spOut = 0;
-			for( int j=0; j<h1.getCols(); j++ ) {
+			for (int j = 0; j < h1.getCols(); j++) {
 				double lsp = (double) h1.cNnz[j] * h2.rNnz[j] / mnOut;
-				spOut = spOut + lsp - spOut*lsp;
+				spOut = spOut + lsp - spOut * lsp;
 			}
-			nnz = (long)(spOut * mnOut);
+			nnz = (long) (spOut * mnOut);
 		}
-		
-		//compute final sparsity
-		return OptimizerUtils.getSparsity(
-			h1.getRows(), h2.getCols(), nnz);
+
+		// compute final sparsity
+		return OptimizerUtils.getSparsity(h1.getRows(), h2.getCols(), nnz);
+
 	}
-	
+
 	private static class MatrixHistogram {
-		private final int[] rNnz;    //row nnz counts
-		private int[] rNnz1e = null; //row nnz counts for cols w/ <= 1 non-zeros
-		private final int[] cNnz;    //column nnz counts
-		private int[] cNnz1e = null; //column nnz counts for rows w/ <= 1 non-zeros
+		private final int[] rNnz; // row nnz counts
+		private int[] rNnz1e = null; // row nnz counts for cols w/ <= 1
+										// non-zeros
+		private final int[] cNnz; // column nnz counts
+		private int[] cNnz1e = null; // column nnz counts for rows w/ <= 1
+										// non-zeros
```

Comment:
Please, refrain from automatic formatting to avoid such formatting issues. Thanks.